common/coat_of_arms/template_lists/colored_emblem_lists.txt

```diff
@@ -761,6 +761,41 @@ colored_emblem_texture_lists = {
 			100 	= "wc_generic_ce_orc_symbol_68.dds"
 			100 	= "wc_generic_ce_orc_symbol_69.dds"
 			100 	= "wc_generic_ce_orc_symbol_70.dds"
+			100 	= "ce_rune_orc_01.dds"
```

Comment:
I would resize this.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/174493929-b9a5032f-2f45-4eac-90c8-b195be3bc4f6.png)

</details>

Reply:
Resized it.
![image](https://user-images.githubusercontent.com/75175790/174605513-113e3053-5b2e-4830-a94b-9adc03493095.png)